src/world/Actor/PlayerSql.cpp

```diff
@@ -696,6 +697,32 @@ bool Sapphire::Entity::Player::loadInventory()
     }
   }
 
+  auto currencyRes = db.query("SELECT storageId, "
```

Comment:
use fmt::format instead here actually

Reply:
I was following the rest of the code in that regard, however I can rewrite most instances that I find if you'd like

Reply:
yeah a lot of that is pretty old - a lot of the db stuff is a bit antiquated and could be fixed but just fix these for now

I won't make you make fundamental changes outside of what bits you've touched, that'd be a dick move lol

Reply:
Works for me. I'll have the fixes published in a few minutes.